### ISSUE TITLE
Nothing Works

### ISSUE THREAD

**Issue by remotevase** [remotevase/SignTP#4]
No console errors.
Problem: Neither 
[WORLD]
nor
[COORD] 
signs work. 

HELP @Taki21 @applqpak  you guys usually fix my dumb mistakes
I looked through the code many times, nothing wrong...

**Comment by remotevase** [comment]
@applqpak you somewhat helped. You prevented a future crash, but this issue still persists

**Comment by applqpak** [comment]
Okay

**Comment by remotevase** [comment]
bump @Taki21 @thelucyclub @TheDragonRing @ImagicalGamer help plz #squad

**Comment by remotevase** [comment]
STILL NO ONE! 
bump @Taki21 @thelucyclub @TheDragonRing @ImagicalGamer @remotevase help plz #squad

**Comment by thelucyclub** [comment]
LOl im looking.

**Comment by remotevase** [comment]
Lol but at least someone's ACTUALLY HELPING! 🏆 @applqpak  🏆 @thelucyclub

**Comment by dragonwocky** [comment]
I've had looks but I don't understand what's wrong.... D:

**Comment by remotevase** [comment]
okay, 🏆 @TheDragonRing

**Comment by Taki21** [comment]
SORRY SO MUCH BUMPS!

**Comment by Taki21** [comment]
EVERYONE IS BUMPING ME @Taki21 @Taki21 @Taki21 NO OMG xD

**Comment by Taki21** [comment]
Really I have so much people LOL

**Comment by remotevase** [comment]
I know, but this should be your top priority (excluding ur own plugins and IM itself):
reason: This issue has been open for 5 whole days, as opposed to the others which were just opened today. I've been waiting for 5 whole days...

**Comment by remotevase** [comment]
@TheDragonRing THIS IS WHAT WE'RE MISSING! In every subfile, we need

```
public function __construct(Loader $plugin){
        $this->plugin = $plugin;
    }
```

**Comment by Taki21** [comment]
Please Dont Blame Me...lol

**Comment by Taki21** [comment]
Something Wrong? New Error!

**Comment by remotevase** [comment]
still doesnt work, but this time I'm not sure if it's because of the seed errors from IM, or the plugin... Everytime I click the sign I get that error that happens because of the seed changes from @hoyn1mc (wow i failed at that name spell)

**Comment by remotevase** [comment]
Im so sad. Does no one like the idea for this plugin or what? Why does no one help?!?!?! They try one fix and leave entirely, or say that they're looking, but in reality, they are just dozing off for a nap :sob: :sob:

**Comment by remotevase** [comment]
Its been 17 days :open_mouth: since this issue was opened. Not a single fix has been found. @Taki21, who claims to be a PHP professor hasn't even helped. What kind of a teacher **is** he?!?!?!

**Comment by Taki21** [comment]
Sorry...Lots of people demand me everyday : (

**Comment by remotevase** [comment]
yes, thats every day. This has been for the past **17** days, not _1_ day...

**Comment by Taki21** [comment]
![image](https://cloud.githubusercontent.com/assets/14851895/15196889/8f60f04c-179c-11e6-87ad-0a017959bf5a.png)
: ( 
![image](https://cloud.githubusercontent.com/assets/14851895/15196928/c0cdafee-179c-11e6-94a8-9da48065bdfc.png)
Anyway... Ill try but please mention me so i can remember to help u

**Comment by remotevase** [comment]
mmmhmmm sure...

**Comment by Taki21** [comment]
![image](https://cloud.githubusercontent.com/assets/14851895/15197026/3f867f46-179d-11e6-8ae7-7eedecf5d1ee.png)
![image](https://cloud.githubusercontent.com/assets/14851895/15197030/445719a4-179d-11e6-83b7-ce20149997c0.png)

**Comment by Taki21** [comment]
U understand?

**Comment by remotevase** [comment]
seems like this situation where Im playing both roles, but mainly the dont give up on me part
Do you actually give up?

**Comment by Taki21** [comment]
lol
